src/test/ui/simd/issue-94382.rs

```diff
@@ -0,0 +1,17 @@
+#![feature(platform_intrinsics)] //~ ERROR module has missing stability attribute
+#![feature(staged_api)]
```

Comment:
I don't think the staged_api feature is needed for the test

Reply:
I tried removing this, it changes the test to only give an error on `#[rustc_const_stable(...)]` saying it can't be used outside stdlib, is this expected?

Reply:
Oops, yea, that is kinda expected. Removing that attribute, too, should work in this test I think.